policy/templates/test-square-ui/.github/workflows/release-tests-ui.yml

```diff
@@ -0,0 +1,20 @@
+name: UI Regresion Tests
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    paths:
+      - 'tests/ui/**'
+  push:
+    branches:
+      - master
+      - release-*
+    paths:
+      - 'tests/ui/**'
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+jobs:
+  {{ $Tests := list "ui"  }}
+  {{ $Branchvals := dict "Features" .Branchvals.Features "Tests" $Tests  }}
+  {{- template "auto-test" (dict "Name" "tyk-analytics-qa-gates" "Branchvals" $Branchvals) }}
```

Comment:
So there is repo called tyk-analytics-qa-gates? It is only set to figure out if the test-square UI tests are being rendered?

If yes, why not just use the test-square-ui feature in the conditionals instead of pretending that there is a repo called t-a-q-g?